Traceback (most recent call last):
  File "/local/293689/main.py", line 176, in <module>
    y_pred, _ = gp.predict(Xu_fixed, y_train)
TypeError: predict() takes 2 positional arguments but 3 were given
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[23045,1],0]
  Exit code:    1
--------------------------------------------------------------------------
cp: missing destination file operand after '*.png/home/yanw/uncertain/results/figures/0617'
Try 'cp --help' for more information.
cp: missing destination file operand after '/home/yanw/uncertain'
Try 'cp --help' for more information.
